client/src/store/modules/eventType/mutations.js

```diff
@@ -0,0 +1,31 @@
+import * as mutations from './types/mutations';
+import { eventTypeMapper } from '@/services/Normalizer';
+
+export default {
+    [mutations.SET_EVENT_TYPES]: (state, eventTypes) => {
+        state.eventTypes = {
+            ...state.eventTypes,
+            ...eventTypes.reduce(
+                (prev, eventType) => ({
+                    ...prev,
+                    [eventType.id]: eventTypeMapper(eventType)
```

Comment:
if you want to store normalized state, with hashmap you  also need an array of ids, so it should look like:

eventTypes: [ 'id1', 'id2', 'id3' ]
eventTypeById: {
  id1: { ... },
  id2: { ... },
  id3: { ... },
}